playerStats.js

```diff
@@ -15,6 +15,16 @@ class PlayerStats
         localStorage.rotationSpeedRad = JSON.stringify(0.02)
         localStorage.repairKitCount = JSON.stringify(0)
 
+        var tanks = [
```

Comment:
Introduce tank class and add static factory method for tanks. Then we have one place were the instances are created and you can use the factory method in PlayerStats to generate the tank instances. You'll probably also need the tank class to reference the mounted weapon (see https://app.asana.com/0/0/1201689130400169/f )